config.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"github.com/ddollar/forego/Godeps/_workspace/src/github.com/subosito/gotenv"
```

Comment:
I think this dependency has already be add to Godeps. In fact i just reuse it. See https://github.com/ddollar/forego/blob/master/env.go#L8